pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/PersistentTopicsTest.java

```diff
@@ -1148,4 +1149,18 @@ public void testAdminTerminatePartitionedTopic() throws Exception{
             Assert.assertEquals(e.getMessage(), "Termination of a non-partitioned topic is not allowed using partitioned-terminate, please use terminate commands.");
         }
     }
+
+    @Test
+    public void testTopicReferenceCommonExceptionWillReturnRestException() {
+        String topicName = "topicName";
+        String exceptionDetail = "exceptionDetail";
+        doThrow(new RuntimeException(exceptionDetail)).when(pulsar).getBrokerService();
```

Comment:
You should when the `pulsar().getBrokerService().getTopicIfExists()`, and return a not complete completableFuture.

Reply:
done.